capitalize_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'decorator'
+class CapitalizeDecorator < Decorator
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end

```

Comment:
- [ ] `correct_name` function should capitalize the first letter of the given name. Therefore should looks like this: 
```ruby
    def correct_name
      @nameable.correct_name.capitalize
    end
```